### PR TITLE
Config files for DC-0 rerun

### PR DESCRIPTION
**Pull request by zonca**
- import TOML files from CHLAT only run
- remove CMB with solar dipole
- rename cmb to cmb lensed
- point to newly exported instrument model
- now mapsims version added automatically
- run all channels and choose automatically modeling_nside
- set modeling_nside for components to avoid ud_grade

### CODE REVIEW

**Comment by zonca** [comment]
@jdborrill I have prepared the configuration files for the single components, the only thing missing is the dipole, that I will add when doing the combined maps.

Can you check for obvious usability issues? for example I renamed `cmb` to `cmb_lensed`

File: 202305_dc0/common.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+lmax_over_modeling_nside = 2.5
+healpix = true
+car = false
+output_folder = "output_lsq/{nside}/{tag}/0000"
+output_filename_template= "cmbs4_{tag}_uKCMB_{telescope}-{band}_nside{nside}_0000.fits"
```
Comment (zonca) [inline 202305_dc0/common.toml:5]: @jdborrill as for file naming, I don't think we want to implement the Data Schema naming here.

**Comment by zonca** [comment]
files for single component have `_high` or `_low` if they are needed for a high or low complexity models, otherwise they are for the Medium complexity model.

**Comment by jdborrill** [comment]
> files for single component have `_high` or `_low` if they are needed for a high or low complexity models, otherwise they are for the Medium complexity model.

Can we explicitly name medium too?

File: 202305_dc0/common.toml (hunk 1 above)
Comment (jdborrill) [reply 202305_dc0/common.toml:5]: Why not?

Comment (zonca) [reply 202305_dc0/common.toml:5]: Data Schema naming is based on component bits, we cannot use it here as we have too many components.
Also the `mapXX` filename component won't be as useful, we only have 1 type of map.
So not much is left of the file naming schema.

Comment (jdborrill) [reply 202305_dc0/common.toml:5]: Fair enough. What is "tag" here?

Comment (zonca) [reply 202305_dc0/common.toml:5]: is the name of the component, for example `dust_medium`

Comment (jdborrill) [reply 202305_dc0/common.toml:5]: Would it be worth using the internal PySM component naming? "high", "medium" and "low" complexity will (i) change with time, and (ii) only apply to the collection of components that define a total model. So I'd only expect us to use them (with version numbers) for the combined maps.

Comment (zonca) [reply 202305_dc0/common.toml:5]: so, here are the components with both names:

```
ame_high.toml:pysm_components_string = "a2"
ame_medium.toml:pysm_components_string = "a1"
cib.toml:pysm_components_string = "cib1"
cmb_lensed.toml:pysm_components_string = "c4"
cmb_unlensed.toml:pysm_components_string = "c3"
co_low.toml:pysm_components_string = "co1"
co_medium.toml:pysm_components_string = "co3"
dust_high.toml:pysm_components_string = "d12"
dust_low.toml:pysm_components_string = "d9"
dust_medium.toml:pysm_components_string = "d10"
freefree_medium.toml:pysm_components_string = "f1"
ksz.toml:pysm_components_string = "ksz1"
radio.toml:pysm_components_string = "rg1"
synchrotron_high.toml:pysm_components_string = "s7"
synchrotron_low.toml:pysm_components_string = "s4"
synchrotron_medium.toml:pysm_components_string = "s5"
tsz.toml:pysm_components_string = "tsz1"
```

Maybe combine the 2, so do `dust_d12`, `dust_d9`? So some of them which are obscure, like `c3`, or `rg1` , become `cmb_c3` and `radio_rg1`.

Comment (jdborrill) [reply 202305_dc0/common.toml:5]: Sounds good.

**Pull request merged**